app.py

```diff
@@ -142,6 +156,882 @@ def viewOptions():
 
 
 
+def addUser():
+    global cur
+    user = {}
+    print("Enter the details of the user:")
+
+    user["name"] = input("Please provide name of the user:")
+    user["email"] = input("Please provide the email id of the user:")
+    user["password"] = input("Please provide the password:")
+    user["address"] = input("Please provide the address:")
+    user["phone"] = input("Please enter the phone number [without space or hyphen]")
+
+    try:
+        query = "INSERT INTO USER VALUES(NULL, '%s','%s','%s','%s',%s, '00:00:00');" %(user["password"],user["name"],user["email"],user["address"],user["phone"])   
+        cur.execute(query)
+        con.commit()
+    except Exception as e:
+        con.rollback()
+        print(e)
+        print("Error: Check your inputs.")
+   
+
+def addProfile():
+    print("Enter the profile details of the user:")
+    profile = {}
+    # try:
+    #     query = "SELECT user_id FROM USER WHERE password='%s' AND name='%s' AND email='%s'" %(user['password'],user['name'],user['email'])
+    #     cur.execute(query)
+    #     profile["user_id"] = cur.fetchone()[0]
+    # except Exception as e:
+    #     cur.rollback()
+    #     print(e)
+    #     print("Something went wrong")
+    profile['user_id'] = input("Enter user id of the user: ")
+    profile['dob'] = input("Enter Date-of-Birth in YYYY-MM-DD format: ")
+    profile['sex'] = input("Enter sex of the use [Male, Female, Others, PreferNotToSay]: ")
+
+    try:
+        query = "INSERT INTO PROFILE VALUES (%s,'%s','%s');" %(profile["user_id"],profile["dob"],profile["sex"])
+        cur.execute(query)
+        con.commit()
+    except Exception as e:
+        con.rollback()
+        print(e)
+        print("Error: Check your inputs.")
+
+def addPost():
+    global cur
+    post = {}
+
+    post['user_id'] = input("Enter the user ID:")
+    # ts = time.time()
+    post['time'] = getCurrentTimeStamp()
+    post['text'] = input("Enter the post text:")
+    post['media'] = input("Enter media associated with the post")
+    
+    try:
+        query = "INSERT INTO POST VALUES (NULL, '%s', '%s', '%s', %s);" %(post["time"], post["text"], post["media"], post["user_id"])
+        cur.execute(query)
+        con.commit()
```

Comment:
In POST, STORIES and, COMMENT either text or media can be null. But the current code, doesn't recognize it as null even if it's an empty string. We have to find a way to overcome this. I'm thinking the following:

Instead of having `'%s'`, we pass text and media to a function that will check if it's supposed to be null, if not it will return `'text'`and `'media'`. I am not implementing this right now before knowing ways to sanitize the input, perhaps we can do this along with sanitizing the input.

Reply:
Not planning to do this in this PR.

Reply:
> @pavani-17 @alapan-sau It's almost done: please review now. I think the SQL file can also merge and that we can clean comments later. Otherwise, I will remove it before merging.

The code seems fine, However, I didn't test all the insertions. I checked the first 5. They are working fine
All updates and deletions are implemented and tested by me and pavani.  Once, u merge this or, we will upload ours. In the mean time, we can start with the functional requirements